projects/core/src/user/store/effects/user-replenishment-orders.effect.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { Actions } from '@ngrx/effects';
+import { provideMockActions } from '@ngrx/effects/testing';
+import { cold, hot } from 'jasmine-marbles';
+import { Observable, of, throwError } from 'rxjs';
+import { ReplenishmentOrderList } from '../../../model/replenishment-order.model';
+import { UserReplenishmentOrderAdapter } from '../../connectors/replenishment-order/user-replenishment-order.adapter';
+import { UserReplenishmentOrderConnector } from '../../connectors/replenishment-order/user-replenishment-order.connector';
+import { UserActions } from '../actions/index';
+import * as fromUserOrdersEffect from './user-replenishment-orders.effect';
```

Comment:
should have replenishment in the name so that when we find it, it wouldn't be the same as order history one

Reply:
or just shorten like all other unit test instead of like fromEffect / fromEffects